lib/pwnlib/tubes/serialtube.rb

```diff
@@ -0,0 +1,113 @@
+# encoding: ASCII-8BIT
+
+require 'rubyserial'
+
+require 'pwnlib/tubes/tube'
+
+module Pwnlib
+  module Tubes
+    # Serial Connections
+    #
+    # Copy & Paste of macro definition from tube.rb because YARD doesn't follow includes.
+    # @!macro [new] raise_eof
+    #   @raise [Pwnlib::Errors::EndOfTubeError]
+    #     If the request is not satisfied when all data is received.
+    #
+    class SerialTube < Tube
+      # Instantiate a {Pwnlib::Tubes::SerialTube} object.
+      #
+      # @param [String] port
+      #   A device name for rubyserial to open, e.g. /dev/ttypUSB0
+      # @param [Integer] baudrate
+      #   Baud rate.
+      # @param [Boolean] convert_newlines
+      #   If +true+, will convert local +context.newline+ to +"\\r\\n"+ for remote
```

Comment:
I guess no escape needed here?
`+"\r\n"+` should be enough

Reply:
And this sentence seems not complete

Reply:
What I try it without the double escape, in the .html files generated by YARD, I see
```If true, will MARKER convert local context.newline to "rn" for remote```
I copied the syntax from tube.rb, which does the same thing. Note that in ``@example`` blocks the rules are different.